### PR TITLE
OBSDOCS-1091: Add a disclaimer sentence about lack of support for undocumented features

### PR DESCRIPTION
**Pull request by max-cx**
Version(s):

4.12, 4.13, 4.14, 4.15, 4.16, 4.17

Issue:

https://issues.redhat.com/browse/OBSDOCS-1091

Link to docs preview:

- https://79459--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr-tracing-rn.html

- https://79459--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-rn.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: observability/distr_tracing/distr-tracing-rn.adoc [hunk 1]
```diff
@@ -6,6 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
```
Comment (dfitzmau) [inline observability/distr_tracing/distr-tracing-rn.adoc:9]: The note breaks the following guideline in the [RH SSG](https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc):
 
Screenshot from 2024-07-30 17-13-43

Comment (dfitzmau) [reply observability/distr_tracing/distr-tracing-rn.adoc:9]: Maybe the note could be moved under the first paragraph in the Red Hat build of OpenTelemetry overview module.

File: snippets/distr-tracing-tempo-required-secret-parameters.adoc [hunk 2]
```diff
@@ -1,7 +1,6 @@
 // Text snippet included in the following modules:
 //
-// * distr-tracing-tempo-install-web-console.adoc
-// * distr-tracing-tempo-install-cli.adoc
+// * distr-tracing-tempo-storage-ref.adoc
```
Comment (dfitzmau) [inline snippets/distr-tracing-tempo-required-secret-parameters.adoc:3]: Not a major change but was this change intended?

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: snippets/distr-tracing-tempo-required-secret-parameters.adoc (hunk 2 above)
Comment (max-cx) [reply snippets/distr-tracing-tempo-required-secret-parameters.adoc:3]: Thank you. Yes, it's just about fixing some outdated metadata.

File: observability/distr_tracing/distr-tracing-rn.adoc (hunk 1 above)
Comment (max-cx) [reply observability/distr_tracing/distr-tracing-rn.adoc]: @dfitzmau, thank you. :sweat_smile: Legally speaking, I'm good because this snippet has been included in assemblies rather than modules.

In any case, I've moved the admonition include statement down the assembly files.

**Review by pavolloffay — APPROVED** [review]

**Comment by opayne1** [comment]
@max-cx do you have a QE review of this PR?

**Review by IshwarKanse — APPROVED** [review]

**Comment by max-cx** [comment]
@opayne1, do you want to complete this review or add the `merge-review-needed` label again?

**Comment by opayne1** [comment]
I can complete! Thanks for confirming the QE review!

**Review by opayne1 — COMMENTED** [review]
/lgtm
